static/docs/chak.md

```diff
@@ -168,21 +168,21 @@ where green is in zugzwang; moving the king to either side allows white to respo
+However. if the defender has one more piece (except for an immobile quetzal) things change dramatically. No matter if the offering is still there or not, there is no way for the divine king to check the defender's king and cut off all squares within the temple it could go in one move, so green can always either move their other piece, or move the king within the temple. The only thing to be concerned about is blundering the other piece, so it's recommended to keep it on the last 4 ranks as the divine king can never get there. Once it's there, the defender can just move (and safely premove) either the piece around the last 4 ranks or the king within the temple, and the attacker is left helpless.
 
 ### Fortresses 
 
-Fortress is set up with a piece defending the altar and every piece being defended. Against such a setup, a lone king is helpless. 
+A fortress is set up with a piece defending the altar and every piece being defended. Against such a setup, a lone king is helpless. 
 
 ![Fortresses](https://github.com/gbtami/pychess-variants/blob/master/static/images/ChakGuide/Fortresses.png)
 
-All those examples are bareboned in a sense that if either side decided to move any of their pieces away from around the temple, they can no longer form a fortress. If both sides have a bareboned fortress like in above diagrams the game is a dead draw. On the other hand, if one side has a bareboned fortress and the other has a fortress plus any extra piece, as long as that extra piece can safely cross the river without being captured and join the attack on the fortress, it's almost always a win for the side with the extra piece.
+All these examples are bareboned in a sense that if either side decided to move any of their pieces away from around the temple, they can no longer form a fortress. If both sides have a bareboned fortress like in the above diagrams, the game is a dead draw. On the other hand, if one side has a bareboned fortress and the other has a fortress plus any extra piece, as long as that extra piece can safely cross the river without being captured and join the attack on the fortress, it's almost always a win for the side with the extra piece.
 
 ### Other defenses
 
 #### Jaguar
 
-In a lone Jaguar vs a Divine King endgame either side can force a draw by repetition
+In a lone Jaguar vs a Divine King endgame, either side can force a draw by repetition...
```

Comment:
The three dots here look weird. Is there a reason for it?

Reply:
The content after is 

> ![JaguarPerpetual](https://github.com/gbtami/pychess-variants/blob/master/static/images/ChakGuide/JaguarPerpetual.png)

...except when the side with the Jaguar can just ignore their Jaguar being under attack and run their King towards the altar like in this position.

Reply:
That's fair.